libs/librepcb/common/graphics/graphicsview.cpp

```diff
@@ -203,6 +203,39 @@ void GraphicsView::zoomAnimationValueChanged(const QVariant& value) noexcept {
  *  Inherited from QGraphicsView
  ******************************************************************************/
 
+bool GraphicsView::event(QEvent* event) {
+    if (event->type() == QEvent::NativeGesture) {
+        QNativeGestureEvent* gesture = dynamic_cast<QNativeGestureEvent*>(event);
+        switch (gesture->gestureType()) {
+          case Qt::BeginNativeGesture:
+          case Qt::EndNativeGesture: {
+            return true;
+          }
+          case Qt::ZoomNativeGesture: {
+            QTransform transform;
+            transform.scale(1+gesture->value(), 1+gesture->value());
+            setTransform(transform, true);
```

Comment:
Why not just call `scale()` instead of doing the transformation stuff manually?

Reply:
Done.